tuf/utils/pkcs8_test.go

```diff
@@ -11,33 +11,7 @@ import (
 )
 
 func getRSAKey() (data.PrivateKey, error) {
-	raw := []byte(`-----BEGIN RSA PRIVATE KEY-----
```

Comment:
These keys and passwords are specifically created for these tests, so it's ok to have them in here

Reply:
it seems no risk here

Reply:
yes, I think this should be ok to close, unless we want to add a comment in files to explain this